app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -1214,6 +1215,46 @@ class PatientRepositoryAndroidTest {
     assertThat(updatedPatient.syncStatus).isEqualTo(PENDING)
   }
 
+  @Test
+  fun saving_bangladesh_national_identifier_for_a_patient_must_work_as_expected() {
+    val patientProfile = testData.patientProfile(syncStatus = DONE, generateBusinessId = false)
+    patientRepository.save(listOf(patientProfile)).blockingAwait()
+
+    val bpPassportCode = UUID.randomUUID().toString()
```

Comment:
Can we change this to better reflect what an actual national ID would look like?

1. Rename `bpPassportCode` to `nationalId`.
2. Instead of a `UUID`, introduce a sample text that looks like a real national ID.

Reply:
@vinaysshenoy I've made these changes, please review again.